src/config/dynamic-theme-fixes.config

```diff
@@ -12820,6 +12820,7 @@ img[src*="/static/media/sonos"]
 .d-flex > .text-break > img
 .d-flex > .flex-grow-1 > div > img
 img[src*="handshake-logo"]
+.d-flex.align-items-center.flex-wrap a img
```

Comment:
What about appending `:not(img[src*="/misc/flags/"] )`?

Reply:
`:not(img[src*="/misc/flags/"]` breaks icons 


![20220813-1660366753](https://user-images.githubusercontent.com/9846948/184469239-5fd6a7ea-cb87-4514-b06f-5611b105d87e.png)
![20220813-1660366771](https://user-images.githubusercontent.com/9846948/184469240-1f2ddacc-30ff-4aea-ac61-05cbba7e9e44.png)

Reply:
Which selector is breaking them? Try to fix that selector instead of manually reverting the filter.

Reply:
You mixed things IMHO.
`.d-flex.align-items-center.flex-wrap a img` is intended to invert directly logos in Web3 section. 

CSS rule is for revert to default generic colors flags. 

Your idea breaks all site (added in new line standalone)

Reply:
> CSS rule is for revert to default generic colors flags.

Is there a filter by the site that is affecting these flags?

Reply:
Mixing with 
```
.d-md-inline.ml-md-3.mt-1.mt-md-0
span.notranslate
```
invert part of text and breaks things visible. My workaround with CSS hack do this properly, but infact it is ugly.

![20220813-1660367704](https://user-images.githubusercontent.com/9846948/184469629-26e06175-4ef3-4965-9e9c-968534e9df36.png)

![20220813-1660367811](https://user-images.githubusercontent.com/9846948/184469656-ad33394a-c127-4933-9a0e-e39f97f7e589.png)